magefiles/magefile.go

```diff
@@ -496,6 +496,15 @@ func (ci CI) setRequiredEnvVars() error {
 			if !strings.Contains(jobName, "rehearse") {
 				os.Setenv(fmt.Sprintf("%s_CATALOG_URL", envVarPrefix), fmt.Sprintf("https://github.com/%s/%s", pr.Organization, pr.RepoName))
 				os.Setenv(fmt.Sprintf("%s_CATALOG_REVISION", envVarPrefix), pr.CommitSHA)
+				os.Setenv("RELEASE_MANAGED_WORKSPACE", constants.ManagedReleaseTeam)
+				os.Setenv("RELEASE_DEV_WORKSPACE", constants.DevReleaseTeam)
+				if os.Getenv("REL_IMAGE_CONTROLLER_QUAY_ORG") != "" {
```

Comment:
what is setting this?

Reply:
It's used in the build PR of push_to_external_registry test case.

Reply:
The environment variables are required to run the release-pipelines suite.

Reply:
Then why is it here instead of in the release files? My understanding is the only reason to put something in the magefile is if it needs to be set with variables related to the PR

Reply:
These values are set here for running the test cases in CI of release-service-catalog. Otherwise, we don't need it.

Reply:
Same reply